tests/resources/test_issue_type_scheme_associations.py

```diff
@@ -0,0 +1,21 @@
+from tests.conftest import JiraTestCase
+
+
+class IssueTypeSchemeAssociationTests(JiraTestCase):
+    def test_scheme_associations(self):
+        allSchemes = self.jira.issue_type_schemes()
+        # there should be more than 1 scheme
+        self.assertGreaterEqual(len(allSchemes), 2)
+        g2g = False
```

Comment:
```suggestion
        is_a_scheme_associated_with_a_project = False
```
I would avoid acronyms or abbreviations like this in variable names, consider more descriptive names that also better confer the meaning of your test.